Controllers/InputModels/NoteInputModel.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using Taskmanager.Data.Entities;
+
+namespace Taskmanager.Controllers.InputModels
+{
+    public class NoteInputModel 
+    {
+        [Required]
+        [StringLength(300, MinimumLength = 1, ErrorMessage = "Note must be between 1 and 300 characters")]
+        public string Body { get; set; }
+    
+        public static Note MapNote(NoteInputModel inputModel)
+        {
+            if (inputModel != null)
+            {
+                return new Note{ Body = inputModel.Body };
+            }
+
+            throw new NullReferenceException();
```

Comment:
Exception without any message is bad practice as you can't identify the issue later